pkgs/top-level/all-packages.nix

```diff
@@ -7134,7 +7134,7 @@ with pkgs;
     llvmPackages = llvmPackages_13;
   };
 
-  conda = callPackage ../tools/package-management/conda { };
+  conda = with python3Packages; toPythonApplication conda;
```

Comment:
I don't think this PR should be modified in any way (though you could add an extra `"bin"` output to `conda`), but it's currently affected by https://github.com/NixOS/nixpkgs/issues/302376

```
❯ nom build github:EricTheMagician/nixpkgs/conda#conda
Finished at 14:59:39 after 0s
❯ grep python result/nix-support/propagated-build-inputs 
/nix/store/ja8g651wkvd6wk572a7bp09l321cm22w-python3.11-archspec-0.2.3 /nix/store/pjzswvshnfy16b47qjchsc8laasgbq6d-python3.11-conda-libmamba-solver-24.1.0 /nix/store/4pi2v0srm49yqbxbm4qqdy5lqkj8j015-python3.11-conda-package-handling-2.2.0 /nix/store/69wnd4xa2m97kf7nh1lb8qs6wfmhxrdg-python3.11-distro-1.9.0 /nix/store/g2ndkfqfj273d0y8g3h11aq3vmjwh04q-python3.11-jsonpatch-1.33 /nix/store/vy23p29z57xd9r7lmpdqhdbqv8rkvw7n-python3.11-packaging-24.0 /nix/store/j6xj0wmm2hlqrz6as08gczdamxml4mcl-python3.11-platformdirs-4.2.0 /nix/store/yqcv2gvxqjw25ypg2vaixdfl6qcsgpna-python3.11-pluggy-1.4.0 /nix/store/r7wv0b2di4w0w32z83k1ixvx7wdj8zzn-python3.11-pycosat-0.6.3 /nix/store/lwz9pmmfsj76vgwbcf6k0k2h1wp4ma4h-python3.11-requests-2.31.0 /nix/store/n7v6fy6yc9h8n5nm6q1f77cm57l50rqf-python3.11-ruamel-yaml-0.18.6 /nix/store/yd5z6lnhsjim5ik01zbjaijnagi2mn5d-python3.11-tqdm-4.66.2 /nix/store/5nankdpz429pn1fv8ij6mkjmzy5f3a9q-python3.11-truststore-0.8.0 /nix/store/gd3shnza1i50zn8zs04fa729ribr88m9-python3-3.11.8
```